sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEAI.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+
+export const HPEAI = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : ['1/2', '1/2'];
+  const rows = ['auto'];
+  const gap = {
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  const areasDefault = [['learn', 'hpeLogo']];
+
+  const areasSmall = [['learn', 'unassigned']];
+
+  const areas = ['xsmall', 'small'].includes(breakpoint)
+    ? areasSmall
+    : areasDefault;
+
+  let padTop: string = 'none';
+  if (['xsmall', 'small'].includes(breakpoint)) {
+    padTop = 'none';
+  } else if (breakpoint === 'medium') {
+    padTop = 'small';
+  } else if (breakpoint === 'large' || breakpoint === 'xlarge') {
+    padTop = 'large';
+  }
+  return (
+    <PageContent
+      // in marketing page they have 80px padding on the sides
+      // cloest we have is 96. As for the top and bottom they have 40px
+      // we have 48px
+      pad={{ vertical: 'large', horizontal: 'xlarge' }}
+      background={
+        ['xsmall', 'small'].includes(breakpoint)
+          ? { color: 'background', dark: true }
+          : {
+              fill: 'horizontal',
+              image: 'url(/marketing-background.png)',
+              dark: true,
+            }
+      }
+      fill="horizontal"
+    >
+      <Grid areas={areas} columns={columns} rows={rows} gap={gap[breakpoint]}>
+        <Box
+          align={['xsmall', 'small'].includes(breakpoint) ? 'center' : 'start'}
+          gap="medium"
+          gridArea="learn"
+        >
+          {/* // this is off a little for the padding on top compared to page. */}
+          <Box pad={{ top: padTop }}>
+            <Text weight="bold" size="small">
+              Artificial Intelligence (AI) solutions
+            </Text>
+          </Box>
+          {/* // this heading is not large enough can we use size or we steering away? */}
+          <Heading margin="none" level="1">
+            Learn how HPE and NVIDIA unlock AL
+          </Heading>
+          <Box gap="large">
+            <Text
+              size={
+                ['xsmall', 'small'].includes(breakpoint) ? 'small' : 'xlarge'
+              }
+            >
+              AI is everywhere, disrupting every industry and creating limitless
+              opportunities. HPE and NVIDIA are collaborating to deliver
+              co-developed solutions to help you accelerate the adoption of
+              generative AI.
+            </Text>
+            {/* // alignement between text and icon is off */}
+            <Button
+              label="HPE ships first NVIDIA Grace Blackwell system"
+              kind="cta-primary"
```

Comment:
Should this just be primary?

```suggestion
              primary
```

Reply:
yeah I wasnt sure it kinda looked like it had a different design to it I can make it `primary` though
<img width="500" alt="Screenshot 2025-03-04 at 4 48 09 PM" src="https://github.com/user-attachments/assets/615bb4b9-9c81-4b41-a0d3-fd56823bc1ea" />

Reply:
Gotcha -- In that case cta-primary and primary both wouldn't achieve the design. The reason I put the suggestion was because "cta-primary" is just "primary" button with a specific HPE icon, but then you're overriding the icon.

If you could just add an inline comment that we don't have a button kind to match the design for this.

Reply:
oh okay I see what you mean I forgot about the `hpe` icon that I was switching out

Reply:
@taysea just updated